src/api/app/views/webui/package/_commit_item.html.erb

```diff
@@ -21,7 +21,9 @@
             committed
         <% end %>
     <% end %>
-    <%= fuzzy_time(Time.at(commit['time'].to_i)) %> (revision <%= commit['rev'] %>)
+    <% time = Time.at(commit['time'].to_i) %>
+    <span title="<%= time.utc %>" class='commit-fuzzy-time'><%= fuzzy_time(time) %></span>
```

Comment:
Also, I don't like rendering UTC. And I think that the seconds are not necessary.

What about something like this:

``` Ruby
time.strftime("%Y-%m-%d %H:%M")
```